[Compiled Graph] Compiled graph documentation V0 (#49115)
ray-project/ray: doc/source/ray-core/compiled-graph/examples/examples.rst

Ray Compiled Graphs Examples
============================

> This section covers application written using Ray Compiled Graphs.

Reviewer: ```suggestion
The following are applications written using Ray Compiled Graphs.
```
